rpm/auditor-priority-plugin.spec

```diff
@@ -29,6 +29,8 @@ rm -rf $RPM_BUILD_ROOT
 %{_bindir}/%{name}
 
 %changelog
+* Thu Nov 17 2023 Benjamin Rottler <benjamin.rottler@physik.uni-freiburg.de> - 0.3.0
```

Comment:
Today is `Fri Nov 17 2023`. Same comment for all `spec` files.

Reply:
Right, fixed. I also udpated the `Changelog.md`, because there the release date was 2023-11-16